lib/discovery.js

```diff
@@ -186,6 +186,9 @@ function mixinDiscovery(MySQL, mysql) {
         (table ? ' WHERE table_name=' + mysql.escape(table) : ''),
       'table_name, ordinal_position', {});
     }
+    if (options.orderBy) {
```

Comment:
Can we add a test for this options param too ?

Reply:
Didn't understand. Can you please explain?